chemclipse/plugins/org.eclipse.chemclipse.msd.model/src/org/eclipse/chemclipse/msd/model/matrix/ExtractedMatrix.java

```diff
@@ -87,62 +87,63 @@ private Boolean checkHighRes( int limit ) {
 				.collect(Collectors.toList()); //
 		return (scans);
 	}
-	
+
 	private int[] getMinMaxMz() {
-		
-		int[] minMaxMz = new int[2];
 
+		int[] minMaxMz = new int[2];
 		Stream<Double> s = scans.stream() //
 				.flatMap(scan -> scan.getIons().stream()) //
-				.map(x -> (Double) x.getIon());
+				.map(x -> (Double)x.getIon());
 		DoubleSummaryStatistics d = s.collect(Collectors.summarizingDouble(value -> value));
-		
 		minMaxMz[0] = AbstractIon.getIon(d.getMin());
 		minMaxMz[1] = AbstractIon.getIon(d.getMax());
-		return (minMaxMz); 
+		return (minMaxMz);
 	}
 
 	public float[][] getMatrix() {
+
 		return this.signal;
 	}
 
 	public int getNumberOfScans() {
+
 		return this.numberOfScans;
 	}
 
 	public int getNumberOfIons() {
+
```

Comment:
the number of ions is equal to the length of the first matrix element (this.signal[0].length) and thus could be ommited.

Reply:
removed method altogether and documented the dimensions of signal array in javadoc instead